tests/Internal/DateTimeStringBugWorkaroundTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+
+namespace ProophTest\EventStore\Internal;
+
+
+use PHPUnit\Framework\TestCase;
+use Prooph\EventStore\Internal\DateTimeStringBugWorkaround;
+
+class DateTimeStringBugWorkaroundTest extends TestCase
+{
+    public function test_it_correctly_converts_datetime_with_missing_micros()
+    {
+        $date = "2020-12-14T12:55:57Z";
+        $expectedConvertedDate = "2020-12-14T12:55:57.000000Z";
+        $convertedDate = DateTimeStringBugWorkaround::fixDateTimeString($date);
+        $this->assertEquals($convertedDate, $expectedConvertedDate);
```

Comment:
iirc expected value is supposed to be the first argument and the actual value the second one, you swapped them